trackers_manager:

  line_tracker_distance:
    default_v_des: 2.0
    default_a_des: 1.0

  line_tracker_min_jerk:
    default_v_des: 2.0
    default_a_des: 1.0
    default_yaw_v_des: 0.8
    default_yaw_a_des: 0.2
